javaee-like-getting-started/src/test/java/io/quarkus/ts/javaee/gettingstarted/MapStructIT.java

```diff
@@ -0,0 +1,55 @@
+package io.quarkus.ts.javaee.gettingstarted;
+
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.restassured.RestAssured;
+
+@QuarkusScenario
+public class MapStructIT {
+
+    @Test
+    public void testInjectionWithJakartaCdiComponentModel() {
+        RestAssured
+                .get("/mapper/injected-jakarta-cdi")
+                .then()
+                .statusCode(200)
+                .body("greeting", Matchers.is("Hello Sebastian from Maximillian"));
+    }
+
+    @Test
+    public void testInjectionWithJakartaComponentModel() {
+        RestAssured
+                .given()
+                .get("/mapper/injected-jakarta-happy-impl")
+                .then()
+                .statusCode(200)
+                .body("partOne.data", Matchers.is("original_happy"))
+                .body("partTwo", Matchers.is("1"));
+        RestAssured
+                .given()
+                .get("/mapper/injected-jakarta-sad-impl")
+                .then()
+                .statusCode(200)
+                .body("partOne.data", Matchers.is("original_sad"))
+                .body("partTwo", Matchers.is("2"));
+        RestAssured
+                .given()
+                .get("/mapper/injected-jakarta-default-method")
+                .then()
+                .statusCode(200)
+                .body("partOne.data", Matchers.is("original_default"))
+                .body("partTwo", Matchers.is("3"));
+    }
+
+    @Test
+    public void testInjectionWithCdiComponentModel() {
```

Comment:
I'm bit confused by method name because I didn't notice any injection for `/mapper/no-cdi` and `SeriousTalkMapper` uses the default componentModel.

```
@Mapper
public interface SeriousTalkMapper {
```
https://github.com/mapstruct/mapstruct/blob/main/core/src/main/java/org/mapstruct/Mapper.java#L158

Reply:
sure, name should be `testInjectionWithDefaultComponentModel` or something like that

Reply:
but where is injection used?

Reply:
there is no injection, it makes sense to test default model in native mode because it works based on static variables and you want to know there is no initialization issue. URL says it `/mapper/no-cdi`, so I just copied previous name.

Reply:
but I can't see `injected-cdi` called, I think I accidentally rewritten it to this "no-cdi", but I remember testing it, I'll try to remember how and add it.

Reply:
done